mava/components/updating/checkpointer.py

```diff
@@ -31,6 +32,7 @@
 @dataclass
 class CheckpointerConfig:
     checkpoint_minute_interval: float = 5
+    restore_best_net: str = "None"
```

Comment:
I would just make this `None` and not the string "None"

Reply:
This is because of a mypy issue but I will change it and write `#type: ignore`

Reply:
I think rather make it a `Union[str, None]` type

Reply:
Nice idea! Done